.github/workflows/run_merge_test.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+name: Running tests
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
we have no master branch, I think you mean main? :)

It should probably also run on PRs like this one?

(see here for example https://github.com/aidotse/PASEOS/blob/a8126c7b231088d27822cfb386b3f617711bd21c/.github/workflows/run_tests.yml#L9 )

Reply:
Yea, I read up a bit and I updated the workflow that we have instead.